### ISSUE TITLE
Is it possible to add an `onComplete` event to FlowBuilder?

### ISSUE THREAD

**Issue by lijy91** [felangel/flow_builder#5]
**Is your feature request related to a problem? Please describe.**
Same as title

**Describe the solution you'd like**
```dart
FlowBuilder<RegisterFlowState>(
  state: RegisterFlowState(),
  onGeneratePages: (state) {
    return [
      MaterialPage<void>(child: RegisterSetNicknameSubPage()),
      if (state.shouldSetGender)
        MaterialPage<void>(child: RegisterSetGenderSubPage()),
    ];
  },
  onComplete: (state) {
    Navigator.of(context).pushReplacement(
      FadeInPageRoute(builder: (_) => HomePage()),
    );
  }
);
```

**Describe alternatives you've considered**
None

**Additional context**
If my current route is the root route, adding `onComplete` can help me go to the next page better

**Comment by felangel** [comment]
Hi @lijy91 👋 
Thanks for opening an issue!

Definitely! I think that's a great idea and is being done as part of #6 👍

**Comment by felangel** [comment]
This is available as part of `v0.0.1-dev.6` 🎉

**Comment by lijy91** [comment]
Great
